stdlib/public/Concurrency/Actor.cpp

```diff
@@ -351,15 +351,14 @@ static void checkIsCurrentExecutorMode(void *context) {
 
   // Potentially, override the platform detected mode, primarily used in tests.
 #if SWIFT_STDLIB_HAS_ENVIRON
-  const char *modeStr = getenv("SWIFT_IS_CURRENT_EXECUTOR_LEGACY_MODE_OVERRIDE");
-  if (!modeStr)
-    return;
-  
-  if (strcmp(modeStr, "nocrash") == 0) {
-    useLegacyMode = Legacy_NoCheckIsolated_NonCrashing;
-  } else if (strcmp(modeStr, "crash") == 0)  {
-    useLegacyMode = Default_UseCheckIsolated_AllowCrash;
-  } // else, just use the platform detected mode
+  if (const char *modeStr =
+          getenv("SWIFT_IS_CURRENT_EXECUTOR_LEGACY_MODE_OVERRIDE")) {
+    if (strcmp(modeStr, "nocrash") == 0) {
+      useLegacyMode = Legacy_NoCheckIsolated_NonCrashing;
```

Comment:
Should set true/false here. useLegacyMode is a bool and this ends up relying on the enum cases being 0 and 1.

Reply:
Will do, thank you